.travis.yml

```diff
@@ -23,33 +23,40 @@ addons:
 before_script:
   - 'npm install benderjs-cli -g'
 
+  # Build CKEditor only for master, major branches and PRs (since every change eventually goes through PR process).
+  - 'RUN_BUILD=0'
+  - 'if [ "$BUILD" = "1" ] && [ "$TRAVIS_PULL_REQUEST" != "false" ]; then RUN_BUILD=1; fi'
+  - 'if [ "$TRAVIS_BRANCH" = "master" ] || [ "$TRAVIS_BRANCH" = "major" ]; then RUN_BUILD=1; fi'
```

Comment:
If I think correctly, this `if` causes to run tests only on built version of CKE on `master` and `major` branches.